app/api/utility.py

```diff
@@ -1,31 +1,106 @@
-"""Constants for federation."""
+"""Constants and utility functions for federation."""
 
 import os
 import re
+import warnings
 
 import httpx
 from fastapi import HTTPException
 
-#  Neurobagel nodes
-NEUROBAGEL_NODES = os.environ.get(
+#  Neurobagel nodes - TODO: remove default value?
```

Comment:
agreed. Same reference to still open issue as above?

Reply:
New issue created to track this:
- https://github.com/neurobagel/federation-api/issues/35